### PR TITLE
HvX plasma availability tweaks and fix

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Edit: Kuro says keep funny free ammo.

Removed the funny free plasma cells from the weapon vendor, I didn't realise req guns already had free ammo for whatever reason?

Made the SMG/rifle a bit more expensive.

Also fixed a bug with the cannon having an incorrect couple of vars set by default
## Why It's Good For The Game
Funny spam bad.
## Changelog
:cl:
balance: Made SMG and rifle more expensive
fix: Fixed plasma cannon starting with an incorrect ammo type set
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -3384,12 +3384,6 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	flame_radius(2, T, burn_duration = 9, colour = "blue")
 	playsound(T, 'sound/weapons/guns/fire/flamethrower2.ogg', 35, 1, 4)
 
-/datum/ammo/energy/plasma/cannon_standard
-	damage = 20
-	penetration = 15
-	sundering = 0.75
-	damage_falloff = 0.75
-
```
Comment (DeltaFire15) [inline code/modules/projectiles/ammo_datums.dm:3392]: ? Why are you deleting this ammo type when it is still used by the cannon's mode list.

Also more balance during freeze woo

**Comment by deathrobotpunch** [comment]
leave your smelly balance changes out of HvH

**Comment by Al-1ce** [comment]
It's not "spam", this is clearly how it was intended to be used.

**Review by TiviPlus — APPROVED** [review]
Someone chain mjp to his table and take away his coh3 until he learned to code doctrines pls

**Pull request merged**